cid/common.py

```diff
@@ -209,26 +209,21 @@ def track(self, action, dashboard_id):
     def deploy(self, **kwargs):
         """ Deploy Dashboard """
 
-        selection = list()
-        for k, dashboard in self.resources.get('dashboards').items():
-            selection.append(
-                questionary.Choice(title=f"{dashboard.get('name')}", value=k)
-            )
-        try:
-            selected_dashboard = questionary.select(
-                "Please select dashboard to install",
-                choices=selection
-            ).ask()
-        except Exception as e:
-            logger.debug(e, stack_info=True)
-            print(f'\nEnd: {e}\n')
-            return
+        selected_dashboard = get_parameter(
+            param_name='dashboard-ressource-id', 
```

Comment:
Can we use the same dashboard-id here?

Reply:
well, dashboard-id is used when it s a quicksight dashboard-id in customer's acc. Here it is the reference to dashboard in ressource.yaml this is why i make this difference. Otherwise user will put "cudos" instead of "CUDOS" etc see ressource.yaml keys vs names vs dashboard-id.

Reply:
This commit changes slightly the logic of dashboard choice. Before it was selected by the keys in dashboard section of ressource.yaml, now dashboard-id is used

https://github.com/aws-samples/aws-cudos-framework-deployment/pull/217/commits/5372e8d5ba3280a53be3360ea448c61306cfd09b